tests/libvmi/vmi.go

```diff
@@ -43,9 +43,9 @@ func New(name string, opts ...Option) *kvirtv1.VirtualMachineInstance {
 	return vmi
 }
 
-// RandName returns a random name by concatanating the given name with a random string.
-func RandName(name string) string {
-	return name + rand.String(5)
+// RandName returns a random vmi name, with the pattern: DefaultVmiName + "-" + rand.String(N)
+func RandName() string {
```

Comment:
I like less the fact that this function no longer accepts a prefix string, in tests with more than one VMI it may be more useful to allow it in the future.
I guess one way to overcome it is to use a variable length arguments to implement a "default" but that is ugly by itself.
I would prefer you use the default constant name in the callers at this stage.